public/locales/en/modules/torrents-status.json

```diff
@@ -1,9 +1,9 @@
 {
   "descriptor": {
     "name": "Torrent",
-    "description": "Displays a list of the torrent which are currently downloading",
+    "description": "Displays a list of torrents from supported Torrent Download clients.",
```

Comment:
Torrent client not torrent download client

Reply:
I based this of how it's named in the docs, docs will have to be changed.

Reply:
Changed in docs too.